src/styles/starts.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+  &--1 :nth-child(-n + 1),
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(-n + 5) {
+    background-image: url('../images/star-active.svg');
```

Comment:
Use single quotes consistently for URLs in the `background-image` property. Here, double quotes are used, while single quotes are used in line 4. It's important to keep consistency throughout the project.